components/Snackbar/src/MDCSnackbarManager.m

```diff
@@ -534,6 +534,13 @@ + (void)setBottomOffset:(CGFloat)offset {
   manager.overlayView.bottomOffset = offset;
 }
 
++ (void)setAlignment:(MDCSnackbarAlignment)alignment {
+  NSAssert([NSThread isMainThread], @"setAlignment must be called on main thread.");
```

Comment:
Nit: I think this can be removed, clients are expected to perform UI work on the main thread.